src/Halogen/Aff/Util.purs

```diff
@@ -6,27 +6,22 @@ module Halogen.Aff.Util
   ) where
 
 import Prelude
-
 import Control.Monad.Aff (Aff, makeAff, runAff)
 import Control.Monad.Eff (Eff)
 import Control.Monad.Eff.Class (liftEff)
 import Control.Monad.Eff.Exception (throwException, error)
 import Control.Monad.Error.Class (throwError)
 import Control.Monad.Except (runExcept)
-
-import Data.Maybe (Maybe(..), maybe)
-import Data.Either (either)
-import Data.Nullable (toMaybe)
-import Data.Foreign (toForeign)
-
 import DOM (DOM)
 import DOM.Event.EventTarget (eventListener, addEventListener)
-import DOM.HTML.Event.EventTypes (load)
 import DOM.HTML (window)
+import DOM.HTML.Event.EventTypes (load)
 import DOM.HTML.Types (HTMLElement, windowToEventTarget, htmlDocumentToParentNode, readHTMLElement)
 import DOM.HTML.Window (document)
-import DOM.Node.ParentNode (querySelector)
-
+import DOM.Node.ParentNode (QuerySelector(..), querySelector)
+import Data.Either (either)
+import Data.Foreign (toForeign)
+import Data.Maybe (Maybe(..), maybe)
 import Halogen.Aff.Effects (HalogenEffects)
```

Comment:
Okay, thanks for the sharp eyes. Out of curiosity, are the differences between your style and psc-ide's documented somewhere? Is it mostly just string comparison (upper-case first) versus alphabetical ordering?

Personally I would sort imports from like most fundamental to most high-level, but I bet that's too subjective and complex to be practical. It would still suggest that I need to fix this though. ;)